lea/clients/__init__.py

```diff
@@ -20,9 +20,13 @@ def make_client(production: bool, wap_mode=False):
         scopes = scopes_str.split(",")
         scopes = [scope.strip() for scope in scopes]
 
+        service_account_info_path = os.environ["LEA_BQ_SERVICE_ACCOUNT"]
```

Comment:
I hope we didn't go too quickly. I see now that in the doc the `LEA_BQ_SERVICE_ACCOUNT` is not expecting a path but a key-value format (a JSON dump). This is why it wasn't working for me, as I assigned the path of the `.json` to the variable